bridges/TheTVDBBridge.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+class TheTVDBBridge extends BridgeAbstract{
+    
+    const MAINTAINER = "Astyan";
+    const NAME = "TheTVDB";
+    const URI = "http://thetvdb.com/";
+    const APIURI = "https://api.thetvdb.com/";
+    const CACHE_TIMEOUT = 43200; // 12h
+    const DESCRIPTION = "Returns latest episodes of a serie with theTVDB api";
+    const PARAMETERS = array(array(
+            'serie_id'=>array(
+                'type'=>'number',
+                'name'=>'ID',
+                'required'=>true,
+            ),
+            'nb_episode'=>array(
+                'type'=>'number',
+                'name'=>'Number of episodes',
+                'defaultValue'=>10,
+                'required'=>true,
+            ),
+        )
+    );
+    const APIACCOUNT = "RSSBridge";
```

Comment:
I am not sure whether we should hard-code the API key here, it might be against their terms of use.